### PR TITLE
Release | Changelog and version updates for 8.4.0 stable release

### PR DESCRIPTION
**Pull request by peterbae**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,25 @@ All notable changes to this project will be documented in this file.
...
+
+### Fixed issues
+- Fixed an issue with fetching column information provides wrong order of columns intermittently [#1348](https://github.com/microsoft/mssql-jdbc/pull/1348)
+- Fixed an issue with spatial datatypes casting error when Always Encrypted is enabled [#1353](https://github.com/microsoft/mssql-jdbc/pull/1353)
+- Fixed an issue with getColumns() not returning correct type for IS_AUTOINCREMENT and IS_GENERATEDCOLUMN against Azure Data Warehouse [#1356](https://github.com/microsoft/mssql-jdbc/pull/1356)
+- Fixed an issue with STAsBinary returning WKB format instead of CLR format [#1364](https://github.com/microsoft/mssql-jdbc/pull/1364)
+- Fixed an issue with allowing non-MSSQL ResultSets to bulk copy DateTimeOffset [#1365](https://github.com/microsoft/mssql-jdbc/pull/1365)
+- Fixed issues identified by SonarQube [#1369](https://github.com/microsoft/mssql-jdbc/pull/1369)
+- Fixed an issue with batch insertion failing when Always Encrypted is enabled [#1378](https://github.com/microsoft/mssql-jdbc/pull/1378)
+
```
Comment (lilgreenbird) [inline CHANGELOG.md:20]: what about executeBatch issue 1334?

Comment (peterbae) [reply CHANGELOG.md:20]: 1334 is a pull request and it got closed without being merged, so I didn't put it in here.

Comment (lilgreenbird) [reply CHANGELOG.md:20]: sorry, I meant to say 1378, which is already there, was confused..

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,25 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 
+## [8.4.0] Stable Release
+### Added
+- Added support for Data Classification v2 [#1338](https://github.com/microsoft/mssql-jdbc/pull/1338) [#1373](https://github.com/microsoft/mssql-jdbc/pull/1373)
```
Comment (ulvii) [inline CHANGELOG.md:8]: Added support for sensitivity ranking when using SQL Data Discovery and Classification.
For consistency with release.txt

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,25 @@ All notable changes to this project will be documented in this file.
...
+- Added public API to return the database compatibility level [#1345](https://github.com/microsoft/mssql-jdbc/pull/1345)
```
Comment (ulvii) [inline CHANGELOG.md:9]: Added 'SQLServerDatabaseMetaData.getDatabaseCompatibilityLevel()' API to return the database compatibility level.

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,6 +3,25 @@ All notable changes to this project will be documented in this file.
...
+- Added support for Azure SQL DNS Caching [#1357](https://github.com/microsoft/mssql-jdbc/pull/1357)
+
+### Fixed issues
```
Comment (ulvii) [inline CHANGELOG.md:12]: Please also add https://github.com/microsoft/mssql-jdbc/pull/1394

**Comment by ulvii** [comment]
Please update https://github.com/microsoft/mssql-jdbc/blob/dev/mssql-jdbc_auth_LICENSE

File: CHANGELOG.md [hunk 5]
```diff
@@ -3,6 +3,26 @@ All notable changes to this project will be documented in this file.
...
+- Fixed an issue with DatabaseMetaData.getColumns() not returning correct type for IS_AUTOINCREMENT and IS_GENERATEDCOLUMN against Azure Data Warehouse [#1356](https://github.com/microsoft/mssql-jdbc/pull/1356)
+- Fixed an issue with Geography.STAsBinary() and Geometry.STAsBinary() returning WKB format instead of CLR format [#1364](https://github.com/microsoft/mssql-jdbc/pull/1364)
+- Fixed an issue with allowing non-MSSQL ResultSets to bulk copy DateTimeOffset [#1365](https://github.com/microsoft/mssql-jdbc/pull/1365)
+- Fixed issues identified by SonarQube [#1369](https://github.com/microsoft/mssql-jdbc/pull/1369)
+- Fixed an issue with batch insertion failing when Always Encrypted is enabled [#1378](https://github.com/microsoft/mssql-jdbc/pull/1378)
+
+### Changed
+- Updated Gradle build file to fix Azure pipelines [#1359](https://github.com/microsoft/mssql-jdbc/pull/1359)
+- Added database name to Always Encrypted enclave caching key [#1388](https://github.com/microsoft/mssql-jdbc/pull/1388)
+- No longer validating certificate expiry date when creating encrypted connection [#1394](https://github.com/microsoft/mssql-jdbc/pull/1394)
```
Comment (ulvii) [inline CHANGELOG.md:24]: We are now validating  both beginning and expiration dates

File: mssql-jdbc_auth_LICENSE [hunk 6]
```diff
@@ -1,5 +1,5 @@
 MICROSOFT SOFTWARE LICENSE TERMS
-MICROSOFT JDBC DRIVER 8.3 FOR SQL SERVER
```
Comment (ulvii) [inline mssql-jdbc_auth_LICENSE:2]: This file is missing the update to section 3

File: CHANGELOG.md [hunk 7]
```diff
@@ -3,6 +3,26 @@ All notable changes to this project will be documented in this file.
...
+- Fixed an issue with DatabaseMetaData.getColumns() not returning correct type for IS_AUTOINCREMENT and IS_GENERATEDCOLUMN against Azure Data Warehouse [#1356](https://github.com/microsoft/mssql-jdbc/pull/1356)
+- Fixed an issue with Geography.STAsBinary() and Geometry.STAsBinary() returning WKB format instead of CLR format [#1364](https://github.com/microsoft/mssql-jdbc/pull/1364)
+- Fixed an issue with allowing non-MSSQL ResultSets to bulk copy DateTimeOffset [#1365](https://github.com/microsoft/mssql-jdbc/pull/1365)
+- Fixed issues identified by SonarQube [#1369](https://github.com/microsoft/mssql-jdbc/pull/1369)
+- Fixed an issue with batch insertion failing when Always Encrypted is enabled [#1378](https://github.com/microsoft/mssql-jdbc/pull/1378)
+
+### Changed
+- Updated Gradle build file to fix Azure pipelines [#1359](https://github.com/microsoft/mssql-jdbc/pull/1359)
+- Added database name to Always Encrypted enclave caching key [#1388](https://github.com/microsoft/mssql-jdbc/pull/1388)
+- Added functionality to validate both beginning and expiration dates when creating encrypted connection [#1394](https://github.com/microsoft/mssql-jdbc/pull/1394)
```
Comment (ulvii) [inline CHANGELOG.md:24]: ...both certificate beginning and expiration dates...

**Pull request merged**